### PR TITLE
Align more closely with Trio RTD configuration

### PR DESCRIPTION
**Pull request by altendky**
Specific points to review:
- Use of Trio logo along with the pytest-trio name

Draft for:
- [x] Seeing it work
- [x] Self review

### CODE REVIEW

**Comment by altendky** [comment]
Kick RTD

**Comment by altendky** [comment]
Kick RTD

**Comment by altendky** [comment]
Kick RTD

File: docs/source/conf.py [hunk 1]
```diff
@@ -81,6 +93,15 @@
...
 release = version
 
+# It would be nicer to make this a .png; literally every browser that
+# supports favicons at all now supports png:
+#     https://caniuse.com/#feat=link-icon-png
+# But sphinx won't let me:
+#     https://github.com/sphinx-doc/sphinx/pull/3715
+# Oh well. 'convert favicon-32.png favicon-32.ico' it is. And it's only 2x
+# bigger...
+html_favicon = "_static/favicon-32.ico"
```
Comment (pquentin) [inline docs/source/conf.py:103]: Sphinx supports PNGs now: https://github.com/sphinx-doc/sphinx/pull/3717

File: docs/source/_static/hackrtd.css [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+}
+
+/* Make .. deprecation:: blocks visible
+ * (by default they're entirely unstyled)
+ */
+.deprecated {
+    background-color: #ffe13b;
+}
+
+/* Add a snakey triskelion ornament to <hr>
```
Comment (pquentin) [inline docs/source/_static/hackrtd.css:15]: This does not appear to work, https://trio.readthedocs.io/en/stable/ has it on the top <hr /> but https://pytest-trio--112.org.readthedocs.build/en/112/ does not

**Review by pquentin — APPROVED** [review]

**Review by pquentin — APPROVED** [review]

File: docs/source/_static/hackrtd.css (hunk 2 above)
Comment (altendky) [reply docs/source/_static/hackrtd.css:15]: Thanks, I see I forgot to copy over `ornament.svg`.  But also, could you share a screenshot of where you see it?  I've looked up what a triskelion is `:]` and checked the contents of the file but I can't find it present at the Trio link.  For example, should I be seeing it here?

image

**Pull request merged**